packages/controller-utils/src/util.ts

```diff
@@ -1,4 +1,3 @@
-import { isValidAddress, toChecksumAddress } from '@ethereumjs/util';
```

Comment:
Nit: can we remove this package from `package.json`

Reply:
Removed